pkg/server/test/list_objects.go

```diff
@@ -69,6 +72,7 @@ func TestListObjectsRespectsMaxResults(t *testing.T, ds storage.OpenFGADatastore
 			maxResults:             2,
 			minimumResultsExpected: 2,
 			allResults:             []string{"repo:1", "repo:2", "repo:3"},
+			useCheckCache:          false,
```

Comment:
Can we omit in favor of brevity where the value is `false`?

Reply:
Having more explicit value makes things more clear though

Reply:
Track via https://github.com/openfga/openfga/issues/933